joomla/joomla-cms
# Fix for fix #36345, TinyMCE, make sure iframe is fully loaded

Pull Request for Issue #36345 for fix #24343 

### Summary of Changes

Make sure the script listen iframe load event only once.

### Testing Instructions
Apply patch, run `npm install`

Create subform with editor (example in Custom HTML):
```xml
<field type="subform" name="subform" multiple="true" label="subform">
  <form>
    <field type="editor" name="text" label="editor" buttons="false"  default="I am a text!"/>
  </form>
</field>
```
Run test in Chrome and Firefox, and Safari (if you have one).

Make sure the Article content editor works correctly.

Then add couple rows in subform and try sort them. Then watch editor content.



### Actual result BEFORE applying this Pull Request
In Firefox the Article content editor realods itself, and not able to edit content.


### Expected result AFTER applying this Pull Request
In all Browsers the content editor works correctly.
Content in subform editor works after sorting.


### Link to documentations
Please select:
